web-playground/e2e/sample-tester.test.ts

```diff
@@ -120,38 +118,37 @@ async function testInputFile(input: string, expected: string, query: string, lan
 }
 
 async function setTextarea(selector: string, text: string) {
-    let textInput = await page.$(selector) ?? fail('Did not find text area');
+    let textInput = await page.$(selector) ?? fail('Did not find text input control');
+    let textAreaSelector = `${selector} textarea`;
 
     // Clear the text area first, otherwise the following doesn't work.
-    await textInput.click({ clickCount: 3 });
+    await textInput.click();
+    await page.keyboard.down('ControlLeft')
+    await page.keyboard.press('KeyA')
+    await page.keyboard.up('ControlLeft')
     await textInput.press('Backspace');
```

Comment:
Out of curiosity: Is there not a better way of clearing the text area, rather than simulating Ctrl+A Backspace? Or does it have to be done this way to trigger the appropriate event?

Reply:
That text editor has an API that can be used from within the app code, but as soon as this runs live in a browser it's just a big mess of HTML and magic. I have not found a better way to clear the text.